### ISSUE TITLE
Early game end suggestion may be incorrect

### ISSUE THREAD

**Issue by yoavst** [yiotro/Antiyoy#62]
**Bug:**
[`possibleWinner()`](https://github.com/yiotro/Antiyoy/blob/e60587568a3719479be12dce0d33f9e129ae3d31/core/src/yio/tro/antiyoy/gameplay/FieldController.java#L311) does not take multiple islands into account. 

**Situation:**
I've been playing a user level with multiple island, in which you need to conquer all the islands to win.
While I was having hard time winning on the last island while doing the right moves on the other islands, I was offered to stop playing and win (the "Win or continue" dialog). 
However, on that very last island, I don't think from the current situation I'll be able to ever win. 

![Example](https://i.imgur.com/3q3yDxX.png)

**Comment by yiotro** [comment]
Hmm, I'm not sure about that, but I think that the idea of this level is that it can't be won by killing all your enemies, but it's enough for 'early win' conditions.

**Comment by yoavst** [comment]
It is possible to kill all your enemies. I was able to win 5 out of 6 in a single game, and win the last island on a new game, so it is definitely possible.

**Comment by subchannel13** [comment]
@yoavst Suppose this was a multiplayer game, if you won on all but one island, had >70% territory, who should be declared a winner?

**Comment by yoavst** [comment]
@subchannel13 I agree in this situation. However, in case of a single player game or specifically a map like this one, the hardness of the level is to be able to focus on all the islands.
